h/h_api/resources/schema/bulk_api/command/create_group_membership.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Individal schema allow us to decide something should match this and then tell the user exactly whats wrong. You can do this with a single schema, but by `anyOf` sections tend to result in unhelpful messages.

This also allows us to re-use this schema conveniently when creating these objects (serialisation) and blow up right away. 

The final reason was if we imagine a future where this action is available as an end-point, this would be the return value. Not nested inside an array.